### ISSUE TITLE
Reset after inactivity [feature request]

### ISSUE THREAD

**Issue by Jip-Hop** [tschiemer/filmothek-kiosk-osx#1]
I realise you made this 3 years ago but it's still very useful.
Actually quite strange your app is the only non-commercial kiosk browser I could find find for MacOS.
The only missing feature for me would be to reset the browser after some minutes of inactivity.

**Comment by tschiemer** [comment]
Hi!

Jes, I made it because I didn't find anything else (after a short search)..

A reset would indeed be a good feature - although it shouldn't be such a big thing, I'm currently lacking the time for it. So anyone's welcome to contribute on that part.

In case you use the kiosk app together with a specific webapp only (ie where the kiosk is locked onto the webapp) you could in principle do this within the webapp itself and wouldn't have to rely on the kiosk app.

For the future it might also be worth looking into a remake based on [Electron's webview](https://electron.atom.io/docs/api/webview-tag/), although I'm not sure if a kiosk-/app-lock-in is possible there. This would also be platform independent, on a more accessible code basis and might provide some useful features out-of-the-box.

**Comment by Jip-Hop** [comment]
Yes there are a lot of webview frameworks around these days. But I don't know if locking them down is possible. That's the good thing about yours.

I did notice that the key combination Option-Command-Esc allows the user to force quit filmothek kiosk.

To work around that I've configured the kiosk to automatically start again as soon as it's closed with launchctl.

Yes if I make the web app myself I can do it in JavaScript. What I'm doing now to reset the kiosk is an applescript configured with launchctl to check the inactivity and force quit filmothek. It will re-launch by itself and reset.

But I still have one problem. When the computer starts up it will automatically launch filmothek. But the screen will turn black, sometimes white. It seems the site won't load the first time... The second time it launches it works after I manually force quit.

**Comment by tschiemer** [comment]
Ah, many thanks for the Option-Command-Esc hint.

Yes, I also noticed that at startup there could appear a blank screen. But I attributed that to the local virtual webserver that wasn't ready yet to satisfy the request. I'll pin it down for fixing.
